<launch>

  <arg name="rviz_conf" default="$(find giraffe_robot)/rviz/config.rviz"/>

  <param name="robot_description" command="$(find xacro)/xacro $(find giraffe_robot)/urdf/giraffe.urdf"/>
  
  <!-- Run your chair marker publisher -->
  <node pkg="giraffe_robot" type="chair_marker_publisher.py" name="chair_marker_publisher" output="screen"/>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher"/>
  <node name="joint_state_publisher_gui" pkg="joint_state_publisher_gui" type="joint_state_publisher_gui"/>

  <node name="rviz" pkg="rviz" type="rviz" args="-d $(arg rviz_conf)"  />
  
</launch>